nasa-gibs/worldview
# WV-2778: Add v2 VIIRS NOAA-20 Deep Blue and Dark Target Aerosol layers

## Description

Fixes WV-2778

Added 5 layers: 		
"VIIRS_NOAA20_Aerosol_Type_Deep_Blue_Best_Estimate",
"VIIRS_NOAA20_Angstrom_Exponent_Deep_Blue_Best_Estimate",
"VIIRS_NOAA20_AOT_Deep_Blue_Best_Estimate",
"VIIRS_NOAA20_Angstrom_Exponent_Dark_Target_Ocean",
"VIIRS_NOAA20_AOT_Dark_Target_Land_Ocean",

## How To Test

1. `git checkout wv-2778-virs-atmo-v2`
2.`npm run build && npm start`
3. Go to [this link](http://localhost:3000/?v=-309.19821557408886,-160.48105016658772,213.3991646707227,110.90395851301591&l=Coastlines_15m,VIIRS_NOAA20_AOT_Dark_Target_Land_Ocean,VIIRS_NOAA20_Angstrom_Exponent_Dark_Target_Ocean(hidden),VIIRS_NOAA20_Angstrom_Exponent_Deep_Blue_Best_Estimate(hidden),VIIRS_NOAA20_AOT_Deep_Blue_Best_Estimate(hidden),VIIRS_NOAA20_Aerosol_Type_Deep_Blue_Best_Estimate(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor&lg=true&t=2023-11-29-T18%3A04%3A30Z), and verify the layers load and that you can take image snapshots.

@nasa-gibs/worldview
